src/cpp/src/model_runner.hpp

```diff
@@ -118,24 +134,63 @@ class ModelRunner {
             total_num_tokens += sequence_group->get_num_scheduled_tokens() * num_sequences;
             total_num_blocks += sequence_group->get_num_blocks() * num_sequences;
             max_context_len_val = std::max(max_context_len_val, sequence_group->get_context_len());
+            for (auto seq: sequence_group->get_running_sequences()) {
+                num_generated_ids += seq->get_generated_len();
+            }
         }
 
         ov::Tensor
             input_ids(ov::element::i64, {total_num_tokens}),
+            inputs_embeds(ov::element::f32, {total_num_tokens, hidden_size}),
             position_ids(ov::element::i64, {total_num_tokens}),
             // PA specific parameters
             past_lens(ov::element::i32, {batch_size_in_sequences}),
             subsequence_begins(ov::element::i32, {batch_size_in_sequences + 1}),
             // block_indices are handled in a special fashion below
             block_indices_begins(ov::element::i32, {batch_size_in_sequences + 1}),
             max_context_len(ov::element::i32, {});
+        
+        ov::Tensor generated_ids_embeds;
+        float *generated_ids_embeds_data;
```

Comment:
```suggestion
        float *generated_ids_embeds_data = nullptr;
```
the same for pointers below

Reply:
Done.